model/src/main/java/org/jboss/pnc/rex/model/Task.java

```diff
@@ -143,6 +143,9 @@ public class Task {
     @Getter(onMethod_ = @ProtoField(number = 14, defaultValue = "false"))
     private Boolean starting;
 
+    @Getter(onMethod_ = @ProtoField(number = 15))
+    private List<String> previousTaskNames = new ArrayList<>();
```

Comment:
I don't think there is a need for this field. There is a field dependencies, which are the direct dependencies.